src/SFA.DAS.Provider.PR.Web/Models/HomeViewModel.cs

```diff
@@ -8,16 +8,46 @@ public class HomeViewModel : HomeSubmitModel
     public string TotalCount { get; }
     public IEnumerable<EmployerPermissionViewModel> Employers { get; }
     public string ClearFiltersLink { get; }
+    public string AddEmployerLink { get; }
 
-    public HomeViewModel(GetProviderRelationshipsResponse source, string clearFiltersLink)
+    public HomeViewModel(GetProviderRelationshipsResponse source, string clearFiltersLink, string addEmployerLink)
     {
         Employers = source.Employers.Select(e => (EmployerPermissionViewModel)e);
         TotalCount = "employer".ToQuantity(source.TotalCount);
         ClearFiltersLink = clearFiltersLink;
+        AddEmployerLink = addEmployerLink;
     }
 }
 
 public class HomeSubmitModel
 {
+    public const string HasCreateCohortPermissionKey = "HasCreateCohortPermission";
+
     public string? SearchTerm { get; set; }
+    public bool HasPendingRequest { get; set; }
+    public bool HasAddApprenticePermission { get; set; }
+    public bool HasNoAddApprenticePermission { get; set; }
+    public bool HasRecruitmentPermission { get; set; }
+    public bool HasRecruitmentWithReviewPermission { get; set; }
+    public bool HasNoRecruitmentPermission { get; set; }
+
+    public Dictionary<string, string> ToQueryString()
+    {
+        Dictionary<string, string> result = new();
+
+        if (!string.IsNullOrWhiteSpace(SearchTerm)) result.Add(nameof(SearchTerm), SearchTerm.Trim());
+
+        if (HasPendingRequest) result.Add(nameof(HasPendingRequest), true.ToString());
+
+        if (HasNoAddApprenticePermission != HasAddApprenticePermission) result.Add(HasCreateCohortPermissionKey, HasAddApprenticePermission.ToString());
+
+        if (HasRecruitmentPermission || HasRecruitmentWithReviewPermission || HasNoRecruitmentPermission)
```

Comment:
It think this needs logic that is if they are all true, it should not be added

Reply:
same comment as above